squid/files/squid.conf

```diff
@@ -75,32 +67,56 @@ http_access deny manager
 #
 # INSERT YOUR OWN RULE(S) HERE TO ALLOW ACCESS FROM YOUR CLIENTS
 #
-
+{% if example_rule %}
 # Example rule allowing access from your local networks.
 # Adapt localnet in the ACL section to list your (internal) IP networks
 # from where browsing should be allowed
 http_access allow localnet
 http_access allow localhost
+{%  endif %}
 
-{% for deny in salt['pillar.get']('squid:deny', []) %}
-http_access deny {{deny}}
```

Comment:
I don't have a problem with these changes, but you should try to maintain backward compatibility when possible. As is, this is going to break people's configs who were using squid:allow/squid:deny